src/test/ui/proc-macro/auxiliary/api/parse.rs

```diff
@@ -1,9 +1,15 @@
 use proc_macro::Literal;
 
 pub fn test() {
+    test_display_literal();
     test_parse_literal();
 }
 
+fn test_display_literal() {
+    assert_eq!(Literal::isize_unsuffixed(-10).to_string(), "- 10");
```

Comment:
This looks like a pre-existing bug - this literal will not round-trip, since "- 10" will result in a parse error. Could you open an issue for this?

Reply:
It's fixed in #87267.